docs/v1/use-cases.md

```diff
@@ -1,96 +1,143 @@
-## V1 Use Cases
+# Service Catalog Use Cases
 
-This document contains the complete list of accepted use-cases for the v1
-version of the service catalog.
+## Catalog Management
 
-## High-Level Use Cases
+1. As a user, I want to be able to register a broker with the Kubernetes service
+   catalog, so that the catalog is aware of the services that broker offers
+2. As a user, I want to be able to update a registered broker so that the
+   catalog can maintain the most recent versions of services that broker offers
+3. As a user, I want to be able to delete a broker from the catalog, so that I
+   can keep the catalog clean of brokers I no longer want to support
 
-These are the high-level, user-facing use cases the v1 service catalog will
-implement.
+* Importing a list of brokers: If I have a remote cluster with a service
+  controller that has n brokers registered, how can I import those into my local
+  cluster?
+* Can I export a list of service brokers and types from my service controller?
+* Is there an auth story for adding service brokers?
```

Comment:
I'm fine striking export/import but service controller to broker auth should probably be covered.

/cc @westonh